CHANGELOG.md

```diff
@@ -33,6 +33,7 @@
 ## [22.05.3] - 2022/06/14
 * [Developer]: Fix bug where users with only group access to a project couldn't view sample details or metadata. See [PR 1313](https://github.com/phac-nml/irida/pull/1313)
 * [Developer]: Fix sql order by bug on admin statistics page that was causing page not to load when `ONLY_FULL_GROUP_BY` enabled. See [PR 1313](https://github.com/phac-nml/irida/pull/1313)
+* [Workflow]: Start the Galaxy Data Library timeout when an upload begins rather than when it is first queued up. See [PR 1337](https://github.com/phac-nml/irida/pull/1337)
```

Comment:
This should be moved to the Unreleased section and not 22.05.3

Reply:
Woops, sorry about that. Fixed in18e1e70d473616a48456153fee07b2a38dcf96bb